tuf-spec.md

```diff
@@ -1117,25 +1121,28 @@ repo](https://github.com/theupdateframework/specification/issues).
   rollback attack.  On the next update cycle, begin at step 5.0 and version N
   of the root metadata file.
 
-  * **5.1.5**. Note that the expiration of the new (intermediate) root metadata
+  * #### **5.1.5**.
+      Note that the expiration of the new (intermediate) root metadata
   file does not matter yet, because we will check for it in step 5.1.9.
 
-  * **5.1.6**. **Set the trusted root metadata file** to the new root metadata
-  file.
+  * #### **5.1.6**. **Set the trusted root metadata file**
+      to the new root metadata file.
 
-  * **5.1.7**. **Persist root metadata.** The client MUST write the file to
+  * #### **5.1.7**. **Persist root metadata.**
+      The client MUST write the file to
   non-volatile storage as FILENAME.EXT (e.g. root.json).
 
-  * **5.1.8**. **Repeat steps 5.1.1 to 5.1.8**.
+  * #### **5.1.8**. **Repeat steps 5.1.1 to 5.1.8**.
 
-  * **5.1.9**. **Check for a freeze attack.** The latest known time MUST be
+  * #### **5.1.9**. **Check for a freeze attack.**
+      The latest known time MUST be
```

Comment:
The nonalignment with the header looks odd... is it necessary here and in other places where it happens, too?

Reply:
@trishankatdatadog Yep. If I remove the early wrapping on this line and accordingly rewrap the paragraph at the new indentation, that results in an illegible diff wherein it appears I've rewritten every single word of the client spec, when it's only a whitespace change. ;) So I figured better to let folks review some slightly odd whitespace that doesn't actually have any visual impact, and then clean up the source wrapping later.